src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-style: normal;
+  font-size: 12px;
+  font-family: "Roboto", serif;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
```

Comment:
Get used to style all elements using classes

Reply:
It's just reset